File: C/imc.c
#include<stdlib.h>
#include<stdio.h>
#include<conio.h>
#include<locale.h>

int main(int argc, char const *argv[]) {
  setlocale( LC_ALL, "Portuguese");

  int peso;
  float imc, altura = 0.0;
  printf("digite aqui seu peso: "); scanf("%d", &peso);
  printf("digite aqui sua altura(com vírgula): "); scanf("%f", &altura);

  imc = peso / (altura*altura);

  printf("\nSeu IMC é: %.2f \n", imc);

  if (imc<17) {
    printf("Está MUITO ABAIXO do peso | Magreza Preocupante.\n\n");
  } else if (imc<=18.49) {
    printf("Está ABAIXO do peso | Magreza .\n\n");
  } else if(imc<=24.9){
    printf("Está com peso NORMAL | Saudável.\n\n");
  } else if(imc<=29.9){
    printf("Está Levemente ACIMA do peso | Sobrepeso.\n\n");
  } else if (imc<=34.9) {
    printf("Está ACIMA do peso | Obesidade Grau I.\n\n");
  } else if (imc<=39.9) {
    printf("Está MUITO ACIMA do peso | Obesidade Grau II (Severa).\n\n");
  } else { // imc 40++
    printf("Está MUITÍSSIMO ACIMA do peso | Obesidade Grau III (Mórbida).\n\n");
  }

  system("pause");

  return 0;
}

File: Python/imc.py
# comentários devem ser 2 espaços após o código e 1 espaço após o '#'
'''
exemplo de
comentário em bloco
com 3 apóstrofes
'''
print("=======================================================")
print("\nCALCULADORA DE IMC\n")

peso = int(input('Digite seu Peso: '))

alturaCm = int(input('Agora digite sua altura(em cm): '))
altura = alturaCm/100  # conversão de cm para Metros

imc = peso/(altura*altura)

print("\nSeu IMC é: %.2f \n" % imc)

if imc < 17:
    print(">> Está MUITO ABAIXO do peso | Magreza Preocupante.\n")
elif imc <= 18.49:
    print(">> Está ABAIXO do peso | Magreza.\n")
elif imc <= 24.9:
    print(">> Está com peso NORMAL | Peso Saudável.\n")
elif imc <= 29.9:
    print(">> Está Levemente ACIMA do peso | Sobrepeso.\n")
elif imc <= 34.9:
    print(">> Está ACIMA do peso | Obesidade Grau I.\n")
elif imc <= 39.9:
    print(">> Está MUITO ACIMA do peso | Obesidade Grau II (Severa).\n")
else:
    print(">> Está MUITÍSSIMO ACIMA do peso | Obesidade Grau III (Mórbida).\n")
print("=======================================================")
